packages/babel-plugin/src/utils/evaluate-expression.tsx

```diff
@@ -141,6 +239,11 @@ export const evaluateExpression = (
     ({ value, meta: updatedMeta } = traverseMemberExpression(expression, updatedMeta));
   } else if (t.isFunction(expression)) {
     ({ value, meta: updatedMeta } = traverseFunction(expression, updatedMeta));
+  } else if (t.isCallExpression(expression)) {
+    ({ value, meta: updatedMeta } = evaluateExpression(
+      expression.callee as t.Expression,
```

Comment:
do we need to cast this? could we check if `callee` is an expression?

```
if (t.isCallExpression(expression) && t.isExpression(expression.callee) {}
```

Reply:
Yeah right 😅